src/applications/vaos/services/healthcare-services/transformers.js

```diff
@@ -0,0 +1,154 @@
+const CodeableConcept = {
+  // Code defined by a terminology system
+  coding: {
+    // The identification of the code system that defines the meaning of the symbol in the code.
+    system: '',
+    // The version of the code system which was used when choosing this code.
+    version: '',
+    // A symbol in syntax defined by the system.
+    code: '',
+    // A representation of the meaning of the code in the system, following the rules of the system.
+    display: '',
+    userSelected: false,
+  },
+  // Plain text representation of the concept
+  text: '',
+};
+
+function mapSecondaryStopCode(clinic) {
+  if (clinic.secondaryStopCode) {
+    const obj = {
+      ...CodeableConcept,
+    };
+    obj.coding.code = clinic.secondaryStopCode;
+
+    return [obj];
+  }
+
+  return [];
+}
+
+/**
+ *
+ */
+export function transformAvailableClinic(facilityId, clinic) {
+  return {
+    id: `var${facilityId}_${clinic.clinicId}`,
+    resourceType: 'HealthcareService',
+    // External identifiers for this item
+    identifier: [
+      {
+        system: 'http://med.va.gov/fhir/urn',
+        // TODO: get clinic id
+        value: `urn:va:healthcareservice:${clinic.siteCode}:${facilityId}:${
+          clinic.clinicId
+        }`,
+      },
+    ],
+
+    // Organization that provides this service
+    // NOTE: Is this the location or the Organization????
+    providedBy: 'Organization/39383',
+
+    // Broad category of service being performed or delivered
+    serviceCategory: { ...CodeableConcept },
+
+    // Specific service delivered or performed
+    serviceType: [
+      {
+        // Type of service delivered or performed.
+        // NOTE: Use for primary stop code
+        type: [
+          {
+            // Code defined by a terminology system
+            coding: {
+              // The identification of the code system that defines the meaning of the symbol in the code.
+              system: '',
+              // The version of the code system which was used when choosing this code.
+              version: '',
+              // A symbol in syntax defined by the system.
+              code: clinic.primaryStopCode,
```

Comment:
I would put the type of care id here, instead of the stop code. They're often the same, but I believe `serviceType is supposed to be an equivalent to type of care, and stop codes go in the `characteristic` array.

You might have to pass that id into the transformer, I don't think it's in the clinic data directly.

Reply:
Done.